AngularSpecificAttributesRemover.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const jestDOMElementSerializer = require('pretty-format').plugins.DOMElement;
+
+const serialize = (node, ...rest) => {
+  const nodeCopy = node.cloneNode(true);
+  Object.keys(nodeCopy.attributes)
+    .map(key => nodeCopy.attributes[key].name)
+    .filter(attributeName => attributeName.includes('ng-reflect'))
+    .forEach(attributeNameToRemove => nodeCopy.attributes.removeNamedItem(attributeNameToRemove));
+
+  return jestDOMElementSerializer.serialize(nodeCopy, ...rest);
+};
+
+const test = val =>
+  isElement(val) && jestDOMElementSerializer.test(val) && serializeTestFn(val);
+
+module.exports = {
+  serialize: serialize,
+  test: test
+};
+
+function serializeTestFn(val) {
+  return Object.keys(val.attributes)
+    .map(key => val.attributes[key].name)
```

Comment:
You can direclty use `Object.values(...).map(value => value.name)` here, as you do not really care for the key.

Reply:
Done